### ISSUE TITLE
createGraphics issue with retina displays issue again but different

### ISSUE THREAD

**Issue by dnassler** [processing/p5.js#397]
The following code shows, incorrectly, a blue square in the upper left part of the the canvas with red otherwise on retina displays (i.e. pixel density is 2) but shows, correctly, a full blue canvas on a regular MAC desktop (pixel density is 1). This is using the latest version of p5:  0.3.9

```
var g
function setup() {
  createCanvas(100,100);
  background(255,0,0); // red
  g = createGraphics(100,100);
  g.background(0,0,255); // blue
  noLoop();
}
function draw() {
  image( g );
}
```

**Comment by saivan** [comment]
I think I might have found one. It seems that in the latest version, calling `pg.clear()` on a retina display will only clear the top left hand corner. It clears exactly one quarter of the total space, which indicates that this is also probably a pixel density issue.

**Comment by Spongman** [comment]
@saivan I believe theclear issue was fixed in #2852 but has not been released yet.
